config/lib/socket.io.js

```diff
@@ -6,11 +6,9 @@ var config = require('../config'),
   fs = require('fs'),
   http = require('http'),
   https = require('https'),
-  cookieParser = require('cookie-parser'),
   passport = require('passport'),
   socketio = require('socket.io'),
-  session = require('express-session'),
-  MongoStore = require('connect-mongo')(session);
+  ExtractJwt = require('passport-jwt').ExtractJwt;
```

Comment:
I'm not. It was left over from my attempts to get SocketIO working with the JWT. I'll remove this.